.github/pull_request_template.md

```diff
@@ -0,0 +1,19 @@
+<!-- 
+Thank you very much for putting in this recipe PR!
+
+If you need help or review please let us know.
```

Comment:
Maybe something like: "This repository is very active, so if you need help with a PR or once it's ready for review, please let the right people know."